doc/ansible/dev/developer_guide.md

```diff
@@ -28,7 +28,7 @@ $ sudo dnf install ansible
 In addition to Ansible, a user must install the [OpenShift Restclient
 Python][openshift_restclient_python] package. This can be installed from pip:
```

Comment:
Sync we upgrade the ansible images to use python3 we updated it to use pip3 in the doc. However, pip or pip3 could be used to install open shift.